fivetran_api/aws_lambda/lambda_api_call.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+from botocore.vendored import requests
+import time
+
+
+def lambda_handler(request, context):
+
+    # Fivetran API URL
+    api_key = request['secrets']['api_key']
+    group_id = request['secrets']['group_id']
+    api_secret = request['secrets']['api_secret']
+
+    url = "http://api.fivetran.com/v1/groups/{group_id}/connectors".format(group_id=group_id)
+
+    # Make a request to the endpoint using the correct auth values
+
+    auth_values = (api_key, api_secret)
+    response = requests.request("GET", url, auth=auth_values)
+
+    timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d %H:%M:%S')
+
+    # Convert JSON to dict
+    response_json = response.json()
+    item_list = response_json['data']['items']
+
+    # These will be put into "insert" in the response
+    result_items = []
+    result_tasks = []
+    result_warnings = []
+
+    # Iterate through the items returned by the API
+    for entry in item_list:
+        entry_item = {}
+        entry_tasks = []
+        entry_warnings = []
+        for k, v in entry.items():
+            if k != "status":
+                entry_item[k] = v
+            elif k == "status":
```

Comment:
This doesn't need to be an `elif`, because if `k != status`, then `else` must be the situation where it does. 

Also, could we perhaps have the `k == 'status'` check first, since that's the one that leads to the meatiest logic? It depends on what you want to communicate here semantically though. If it's more natural for your target audience to consider `k != 'status'`, then we can leave it as is.

Reply:
The general case is k != 'status, so I'll keep the order but exchange the `elif` for `else`